_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -71,25 +86,25 @@ With the release of ROS 2 Galactic, we would like to share the new features only
     <tr>
       <td><a href="https://github.com/ros-planning/moveit2/pull/488" target="_blank">Hybrid Planning</a> (global + local planners)</td>
       <td class="not">✕</td>
-      <td class="done">✓</td>
+      <td class="done">pending</td>
     </tr>
     <tr>
-      <td>Realtime Support via DDS</td>
+      <td>Realtime Capable via DDS</td>
       <td class="not">✕</td>
       <td class="done">✓</td>
     </tr>
     <tr>
       <td>Improved Cross-Platform Support</td>
-      <td class="not">✕</td>
+      <td class="dev">via RoboStack</td>
       <td class="done">✓</td>
```

Comment:
Have you used MoveIt2 on Windows?  Are we now supporting people using Windows?  Have we merged the PR so it builds on Windows?

Reply:
The description is that making MoveIt-based applications work on something besides Ubuntu is much easier now (improved) because the underlying ROS 2 infrastructure does support Windows (and Mac to an extent)

Reply:
My problem is mostly with the word "support".  That would seem to imply that we are helpful to people who try to use moveit2 on Windows.  We haven't even merged the PR so that it builds on Windows.

Reply:
I guess it is improved in that we use ros2 but to place a checkmark next to supported seems a bit much.

Reply:
what if we said `Improved Cross-Platform Operation`

Reply:
This is the only one I'm not sure about how to fix

Reply:
The Ubuntu distributions do not belong in this comparison. Giving only Ubuntu versions would even bias cross-platform statements further.

Giving the redundant "based on ROS 2" point in this table again is also not helpful, and the obvious conflict between the two lines brings up more questions than necessary.

I would simply go with your second row only (both in yellow):
```
Builds natively on Windows:
MoveIt 1: via RoboStack
MoveIt 2: under development (link to PR)
```

Reply:
Good point on the ubuntu versions as there are many more targets that are also supported as a result or each ros version.